.github/contributing.md

```diff
@@ -1,4 +1,25 @@
-# Contributing
+# Contributing to Renovate
+
+Thank you for considering contributing to Renovate.
+We appreciate your help!
+Read this guide to get started, and to make sure your contributions are effective.
+
+## Getting Started
+
+### Prerequisites
+
+Read the [prerequisites](https://github.com/renovatebot/renovate/blob/main/docs/development/local-development.md#prerequisites) section of the local development docs.
+
+### Setting up the development environment
+
+Read the [Fork and Clone section](https://github.com/renovatebot/renovate/blob/main/docs/development/local-development.md#fork-and-clone) of the local development docs.
+
+## Submitting Changes
+
+If you've never worked with Git before, please read chapter one, two and three of the [ProGit 2 book](https://git-scm.com/book/en/v2) first.
+
+Then read [ProGit 2, chapter 6.2 GitHub - Contributing to a Project].
+
```

Comment:
We host this project on _Git_ Hub, I don't think we need to go this far down the stack. Else you could also argue we need to start explaining the english language, use of computers, etc.

Reply:
```suggestion
```

Let's get rid of this then. 😄